internal/plugins/helm/plugin.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helm
+
+import (
+	"sigs.k8s.io/kubebuilder/pkg/model/config"
+	"sigs.k8s.io/kubebuilder/pkg/plugin"
+)
+
+// todo: the pluginName and version needs to defined when the config will be implemented
+const pluginName = "helm" + plugin.DefaultNameQualifier
+
+var (
+	supportedProjectVersions = []string{config.Version3Alpha}
+	pluginVersion            = plugin.Version{Number: 1, Stage: plugin.AlphaStage}
+)
+
+var (
+	_ plugin.InitPluginGetter = Plugin{}
+)
+
+type Plugin struct {
+	initPlugin
+}
+
+func (Plugin) Name() string                       { return pluginName }
+func (Plugin) Version() plugin.Version            { return pluginVersion }
+func (Plugin) SupportedProjectVersions() []string { return supportedProjectVersions }
```

Comment:
@joelanford since we use the kb impl we cannot raise a panic here. However, the PROJECT file will not have the plugin info and a todo was added to make clear that the version and name needs to b discussed when we impl the config. See;

```go
$ cat PROJECT 
version: 3-alpha
```

Reply:
I will move by adding the config as well. See; https://github.com/operator-framework/operator-sdk/pull/3193/files#r438958286